common.sh

```diff
@@ -351,7 +354,7 @@ parse_txt_junit_xml()
             sed -i "1s/\(${instance_ip_or_id[1]} [+]\+ \)*\(.\+\)/${instance_ip_or_id[1]} + \2/g" ${file}
         else
             parse_fabtests ${file}
-            break
+            continue
```

Comment:
Was this a bugfix?

Reply:
Yes, I mentioned it in the commit log. A break here will cause the parsing to exit too soon.